zookeeper/component.py

```diff
@@ -180,7 +180,9 @@ def __getattr__(self, name):
         # parent if possible.
         if name in self.__component_annotations__ and self.__component_parent__:
             return getattr(self.__component_parent__, name)
-        raise AttributeError
+        raise AttributeError(
+            f"Component {self.__class__.__name__} does not have any attribute {name}."
```

Comment:
```suggestion
            f"Component {self.__component_name__} does not have any attribute {name}."
```

Do you mind if we use the component name instead?